### PR TITLE
Support Xcode 8/Swift 2.3

### PR DESCRIPTION
**Pull request by ikesyo**
~~I'd like to get Curry v2.3.3 with the fix of https://github.com/thoughtbot/Curry/pull/31.~~ Done.

### CODE REVIEW

**Comment by ikesyo** [comment]
😲 

> xcodebuild: error: Unknown build action 'buildfortesting testwithoutbuilding'.

File: script/cibuild [hunk 1]
```diff
@@ -11,4 +11,4 @@ my @args = ("-workspace", $WORKSPACE, "-scheme", $SCHEME, $ACTION, "CODE_SIGNING
 unshift @args, "-sdk", "iphonesimulator" if $buildSettings =~ /\bPLATFORM_NAME = iphoneos/i;
 
 print "xcodebuild @args\n";
-exec("xcodebuild", @args);
+exec("xcodebuild @args");
```
Comment (ikesyo) [inline script/cibuild]: https://github.com/mdiep/Tentacle/pull/37#issuecomment-234604638: I'm not good at Perl and not sure if this is a right way to handle `XCODE_ACTION="build-for-testing test-without-building"` (`$ACTION`).

File: script/cibuild [hunk 2]
```diff
@@ -8,7 +8,10 @@ my $buildSettings = qx{xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sho
...
 
-unshift @args, "-sdk", "iphonesimulator" if $buildSettings =~ /\bPLATFORM_NAME = iphoneos/i;
+if ($buildSettings =~ /\bPLATFORM_NAME = iphoneos/i) {
+  unshift @args, "-destination", "\"name=iPhone 6s\"";
+  unshift @args, "-sdk", "iphonesimulator";
+}
 
 print "xcodebuild @args\n";
-exec("xcodebuild", @args);
+exec("xcodebuild @args");
```
Comment (ikesyo) [inline script/cibuild]: https://github.com/mdiep/Tentacle/pull/37#issuecomment-234604638: I'm not good at Perl and not sure if this is a right way to handle `XCODE_ACTION="build-for-testing test-without-building"` (`$ACTION`).

**Comment by mdiep** [comment]
Thanks for working on this, @ikesyo! I really appreciate you updating all these projects for Swift 2.3!

**Pull request merged**